share/completions/blender.fish

```diff
@@ -1,10 +1,10 @@
 function __blender_list_scenes -a file
-    if command -q stdbuf
-        stdbuf --output=0 blender --background test.blend --python-expr 'import bpy
+    blender --background $file --python-expr 'import bpy
 
 for name in [scene.name for scene in list(bpy.data.scenes)]:
-    print(name)' | head --lines=-2 | tail --lines=+6
-    end
+    print(f"\t{name}")' |
+        string match -e -r '^\s+' |
+        string replace -r '^\s+' ''
```

Comment:
`string replace -rf`, then you can remove the `match`.

The `-f` is short for `--filter`, and tells `string replace` to only let things through where it needs to replace, so it's short for *exactly* this kind of `match | replace` pipeline.